programmers/난이도별/level02/숫자의_표현/HyeonJeong.py

```diff
@@ -0,0 +1,14 @@
+import math
+def solution(n):
+    answer = 0
+    for i in range(1, math.ceil(n/2)):
+        total = 0
+        for j in range(i, n):
+            total += j
+            if total >= n:
+                if total == n:
+                    answer += 1
+                break
```

Comment:
```suggestion
            if total == n: answer += 1
            if total > n: break
```
이렇게 나눠주는게 좋을 것 같아요!
중첩 반복을 줄이긴 힘들지만 중첩 if는 쉽게 줄일 수 있답니다~